src/ctest.ts

```diff
@@ -362,9 +362,11 @@ export class CTestDriver implements vscode.Disposable {
     private readonly _resultsChangedEmitter = new vscode.EventEmitter<BasicTestResults | null>();
     readonly onResultsChanged = this._resultsChangedEmitter.event;
 
-    async runCTest(driver: CMakeDriver): Promise<number> {
-        log.showChannel();
-        this._decorationManager.clearFailingTestDecorations();
+    public async runCTest(driver: CMakeDriver, customized: boolean = false, testPreset?: TestPreset): Promise<number|null> {
```

Comment:
I remembered that we need that customized flag because there are two ways we customize the test, one is using a test preset, and one is without testpreset based on settings.